_data/projects/sel4.yml

```diff
@@ -1151,5 +1151,5 @@ roadmap:
     display_name: seL4 Microkit example system
     description: A non-trivial example system built using Microkit with various drivers and OS services
     assigned: UNSW
-    status: "Ongoing. A prototype will be released at the end of 2023, with the full release in March 2024."
+    status: "Ongoing. An initial open-source release in March 2024 with implementation and documentation."
```

Comment:
It's now not a full sentence any more. If that was on purpose that's fine from my side. Just pointing it out in case it was not.

Reply:
No it was me just rushing it, sorry.